atlp-rwanda/atlp-devpulse-fn
# Fixing job application

### **PR Description**
Fizing job application process

### **Description of tasks that were expected to be completed**

1. Users should be able to apply for a job
2. The application page should be available

### _**How can it be tested**_

- Clone the repository and install packages using npm install.
- Run the app using npm run dev and make sure you have cloned the backend too [here](https://github.com/atlp-rwanda/atlp-devpulse-bn/pull/180)
